src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
```

Comment:
```suggestion
```

not fixed. 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/729c3c5a-890b-448c-8f14-a84f1d55ec2b)